### PR TITLE
Feat: Added Teams Details  LHS UI part 

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR i have created the LHS part of teams and the routing functionality code refactoring  is remaining 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

File: src/packages/@app/pages/Teams/Teams.ViewModel.ts [hunk 1]
```diff
@@ -28,4 +38,68 @@ export class TeamsViewModel {
...
+  public get workspaces() {
+    return this.workspaceRepository.getWorkspaces();
+  }
+
+  get activeWorkspace() {
+    return this.workspaceRepository.getActiveWorkspace();
+  }
+
+  public get collection() {
+    return this.collectionRepository.getCollection();
```
Comment (Astitva877) [inline src/packages/@app/pages/Teams/Teams.ViewModel.ts:51]: add jsdoc in these file

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**